agent/definitions.py

```diff
@@ -53,7 +53,7 @@ class VulnerabilityMetadata:
 
     title: str
     description: str
-    reference: str
+    reference: str | list[str]
```

Comment:
Sorry but this is not clean, we simply need to switch to references of list[str] only and changes all the other code.